helm_deploy/hmpps-interventions-service/templates/_envs.tpl

```diff
@@ -19,6 +19,33 @@ env:
   - name: INTERVENTIONSUI_BASEURL
     value: "{{ .Values.env.INTERVENTIONSUI_BASEURL }}"
 
+  - name: COMMUNITYAPI_BASEURL
+    value: "{{ .Values.env.COMMUNITYAPI_BASEURL }}"
+
+  - name: COMMUNITYAPI_TEMPORARYREFERRALREQUEST_PROBATIONAREACODE
```

Comment:
naming is incorrect on these -> contactNotificationContext,

same below

Reply:
Fixed